css/style.css

```diff
@@ -118,8 +119,8 @@ p {
 
 .btn-primary:hover,
 .btn-success:active:focus {
-    background-color: #f926b0!important;
-    border-color: #f926b0!important;
+    background-color: #0da5ce!important;
+    border-color: #0da5ce!important;
```

Comment:
Why the `important!` flag? I feel like we're over-engineering with this. Should be clean Bootstrap.